### PR TITLE
8254081: java/security/cert/PolicyNode/GetPolicyQualifiers.java fails due to an expired certificate

### PR DESCRIPTION
**Pull request by rhalade**
/issues add 8254081

---------
### Progress
- [x] Change must not contain extraneous whitespace
- [x] Commit message must refer to an issue
- [x] Change must be properly reviewed

### Issue
 * [JDK-8254081](https://bugs.openjdk.java.net/browse/JDK-8254081): java/security/cert/PolicyNode/GetPolicyQualifiers.java fails due to an expired certificate

### Reviewers
 * [Sean Mullan](https://openjdk.java.net/census#mullan) (@seanjmullan - **Reviewer**) ⚠️ Review applies to f8ff3bbc5e05b1987c4ee74ee767323603b23757
 * [Xue-Lei Andrew Fan](https://openjdk.java.net/census#xuelei) (@XueleiFan - **Reviewer**)

### Download
`$ git fetch https://git.openjdk.java.net/jdk pull/528/head:pull/528`
`$ git checkout pull/528`

### CODE REVIEW

**Comment by rhalade** [comment]
/summary Perform backdated validation of test certificate.

File: test/jdk/java/security/cert/PolicyNode/GetPolicyQualifiers.java [hunk 1]
```diff
@@ -52,6 +53,8 @@ public static void main(String[] args) throws Exception {
         PKIXParameters params = new PKIXParameters(trustAnchors);
         params.setPolicyQualifiersRejected(false);
         params.setRevocationEnabled(false);
+        // Certificates expired on Oct 6th, 2020
+        params.setDate(new Date("July 01, 2020"));
```
Comment (XueleiFan) [inline test/jdk/java/security/cert/PolicyNode/GetPolicyQualifiers.java:57]: The Date() constructor is deprecated, would you like to use the replacement method DateFormat.parse()?

**Review by seanjmullan — APPROVED** [review]

File: test/jdk/java/security/cert/PolicyNode/GetPolicyQualifiers.java (hunk 1 above)
Comment (rhalade) [reply test/jdk/java/security/cert/PolicyNode/GetPolicyQualifiers.java]: I have fixed it, thanks!

**Review by XueleiFan — APPROVED** [review]

**Pull request closed without merging**